pypoint/__init__.py

```diff
@@ -124,7 +124,7 @@ def is_authorized(self):
 
     def user(self):
         """Update and returns the user data."""
-        return self._request(MINUT_USERS_URL)
+        return self._request(MINUT_USERS_URL + "/" + self._token['user_id'])
```

Comment:
```suggestion
        return self._request(MINUT_USERS_URL + "/{}".format(self._token['user_id']))
```
Please use format stings.